docs/deploy/howto-register-aad-applications.md

```diff
@@ -1,21 +1,141 @@
+those App Registrations for you by default. But they both require user to have `Administrator` role in the AAD
+for deployment to succeed. In cases when the user does not have the `Administrator` role, the process of
+deployment can be separated into 2 distinct states:
 
-## AKS
+* creation and setup of App Registrations in AAD tenant
+* creation of Azure resources and deployment of the Azure Industrial IoT platform components
 
-TBD
+With this separation, the first step can be performed by an IT administrator. Deployments will output details
+of App Registrations which can then be passed to a user for running the second step.
+
+## PowerShell scripts
+
+`deploy.ps1` PowerShell script calls `aad-register.ps1` for creating the following App Registrations:
+
+* `<application-name>-service`: used for providing identity for back-end microservices
+* `<application-name>-client`: used for authentication of native clients, such as CLI tool
+* `<application-name>-web`: used for authentication of Web clients, such as Swagger
+
+Use the following command to run `aad-register.ps1` script for creation of App Registrations.
+Specify desired application name instead of `<application-name>` and your tenant id.
+Follow script commands and provide additional details where needed.
+
+> Note: `ReplyUrl` has the following format `https://<application-name>.azurewebsites.net/`, as we are using
+> an instance of App Service to host Engineering Tool.
+
+```bash
+cd deploy/scripts
+./aad-register.ps1 -Name <application-name> -ReplyUrl https://<application-name>.azurewebsites.net/ -TenantId XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX -Output aad.json
+```
+
+The output of the script is a JSON file containing the relevant information to be used as part of the
+deployment and must be passed to the `deploy.ps1` script in the same folder using `-aadConfig` argument.
+
+Use the following command to deploy Azure Industrial IoT platform with pre-created App Registrations.
+Follow script commands and provide additional details where needed.
+
+```bash
+./deploy.ps1 -aadConfig aad.json -applicationName <application-name>
+```
+
+You can find all configuration options for `deploy.ps` [here](./howto-deploy-all-in-one.md#deployment-script-options).
+
+## Microsoft.Azure.IIoT.Deployment
+
+Find links for downloading executable `Microsoft.Azure.IIoT.Deployment` application [here](./howto-deploy-aks.md#download-microsoftazureiiotdeployment-binaries).
+
+`Microsoft.Azure.IIoT.Deployment` application creates the following App Registrations:
+
+* `<application-name>-service`: used for providing identity for back-end microservices
+* `<application-name>-client`: used for authentication of native and Web clients, such as CLI tool or Swagger
+* `<application-name>-aks`: used for providing identity for AKS cluster
+
+Use the following command to run `Microsoft.Azure.IIoT.Deployment` application for App Registration creation
```

Comment:
missing "the"

Reply:
Where ? `the  Microsoft.Azure.IIoT.Deployment application` or `the App Registration` ?

Reply:
the Microsoft.Azure.IIoT.Deployment application

Reply:
Fixed.